.github/workflows/run-pytest.yml

```diff
@@ -0,0 +1,55 @@
+name: Run Unit Tests
+
+on:
+  # trigger on pull requests
+  pull_request:
+
+  # trigger on all commits to master
+  push:
+    branches:
+      - 'master'
+
+  # trigger on request
+  workflow_dispatch:
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
Why no testing on windows?

Reply:
Windows is not fully supported by signac-flow. We have a number of features like command line operations and “with job” that inject UNIX-specific things like trap commands. About 80% of our test suite passes on Windows, if I recall correctly, but tests of job submission and a few other categories fail.

Reply:
This is an area for improvement — we should mark some tests as skip on Windows, and/or block those features with explicit errors on Windows. I don’t think we have received many complaints about support, but I know we have some users of the package’s partial functionality on Windows so I don’t want to explicitly disallow installation of the package on Windows as an unsupported platform.

Reply:
I agree that we can improve here, but I think we are okay to merge this without it.